packages/platform-browser/src/dom/events/event_manager.ts

```diff
@@ -32,7 +32,7 @@ export class EventManager {
    * Initializes an instance of the event-manager service.
    */
   constructor(@Inject(EVENT_MANAGER_PLUGINS) plugins: EventManagerPlugin[], private _zone: NgZone) {
-    plugins.forEach(p => p.manager = this);
+    plugins.forEach((p) => (p.manager = this));
```

Comment:
Just an extra improvement that we can make here: use full name and use `{ ... }` for readability:
```suggestion
    plugins.forEach((plugin) => { plugin.manager = this; });
```